CHANGELOG.md

```diff
@@ -0,0 +1,56 @@
+# Changelog
+
+## Version 2.0.0
+
+### Changes to the SAML integration
+
+#### Defaults to the SHA-256 hash signature hash
+
+The module are now by default using the SHA-256 instead of the deprecated
```

Comment:
Grammar issue here "the module are"? ;-) Maybe something like "This module is now using SHA-256 hashing algorithm by default"